src/test/setup_common.cpp

```diff
@@ -196,3 +196,37 @@ CBlock getBlock13b8a()
     stream >> block;
     return block;
 }
+
+CMutableTransaction BuildCreditingTransaction(const CScript& scriptPubKey, int nValue)
```

Comment:
I don't think `setup_common.cpp` is the right place for this. Please define a new cpp/h header pair in `src/test` for transaction utils, or something like that.

Reply:
Thanks for the feedback, done.